readme.md

```diff
@@ -9,44 +9,67 @@ Hosted version is available at https://yearn.vision.
 
 You will need:
 
-- Erigon for querying historical data
-- Victoria-metrics to pull the metrics, persist them and make them queryable
-- Grafana if you want to set up custom dashboards and alerts
-- Etherscan API key
-- docker and docker-compose (not mandatory but easier usage, see below)
+- Etherscan API key (and API keys for other networks that you want to use)
+- [Docker](https://www.docker.com/) and [Docker Compose](https://github.com/docker/compose
 
 ## Usage
 
-### Prometheus exporter
+Run `make up` to start the exporter.
+
+### Grafana Dashboard & Exporters
 
 ```bash
-# full info
-brownie run exporter
-# realtime tvl only
-brownie run exporter tvl
+# Grafana
```

Comment:
should we just make a .env.example file for this stuff? would be cleaner

Reply:
yes this would be great! I can move to a .example file